src/arch/riscv/isa.cc

```diff
@@ -331,6 +334,60 @@ ISA::setMiscReg(int misc_reg, RegVal val)
         warn("Ignoring write to %s.\n", CSRData.at(misc_reg).name);
     } else {
         switch (misc_reg) {
+
+          case MISCREG_PMPCFG0:
+          case MISCREG_PMPCFG2:
+            {
```

Comment:
Not sure if the spacing is right on this line.

Reply:
Fixed this.